ui/page/dexclient/add_dex_widget.go

```diff
@@ -0,0 +1,182 @@
+package dexclient
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"decred.org/dcrdex/client/asset"
+	"decred.org/dcrdex/client/core"
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/modal"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const testDexHost = "dex-test.ssgen.io:7232"
+
+type addDexWidget struct {
+	*load.Load
+	addDexServer     decredmaterial.Button
+	dexServerAddress decredmaterial.Editor
+	isSending        bool
+	cert             decredmaterial.Editor
+	createWalletWdg  *dexCreateWalletWidget
+	showCreateWallet bool
+}
+
+func newAddDexWidget(l *load.Load) *addDexWidget {
+	dwg := &addDexWidget{
+		Load:             l,
+		dexServerAddress: l.Theme.Editor(new(widget.Editor), "DEX Address"),
+		addDexServer:     l.Theme.Button("Submit"),
+		cert:             l.Theme.Editor(new(widget.Editor), "Cert content"),
+		createWalletWdg:  newDexCreateWalletWidget(l),
+	}
+
+	dwg.addDexServer.TextSize = values.TextSize12
+	dwg.addDexServer.Background = l.Theme.Color.Primary
+
+	dwg.dexServerAddress.Editor.SetText(testDexHost)
```

Comment:
Only applicable for testnet instances:

```suggestion
	if l.WL.MultiWallet.NetType() == dcrlibwallet.Testnet3 {
		dwg.dexServerAddress.Editor.SetText(testDexHost)
	}
```